mysite/tests/services/AddonBuilderTest.php

```diff
@@ -135,9 +135,11 @@ public function testRewriteRelativeLinksAndImages()
         $expected = <<<HTML
 <h1>Heading</h1>
 
-<p><a href="https://github.com/silverstripe/silverstripe-framework/blob/master/relative">Relative</a> and <a href="//absolute.com">absolute</a>.</p>
+<p><a href="https://github.com/silverstripe/silverstripe-framework/blob/master/relative">Relative</a> and
+ <a href="//absolute.com">absolute</a>.</p>
```

Comment:
The whitespace here will cause the tests to break, probably. Instead you could tell phpcs to ignore these lines: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file

Reply:
Now, that is handy!